js-runner/webpack.common.js

```diff
@@ -11,33 +11,37 @@ const common = {
           loader: "babel-loader",
           options: {
             presets: [],
-            plugins: ["transform-object-rest-spread"],
+            plugins: ["@babel/plugin-proposal-object-rest-spread"],
           },
         },
       },
     ],
   },
-  plugins: [
-    new webpack.DefinePlugin({
-      __DEBUG: JSON.stringify(false),
-    }),
-  ],
 };
 
 const browserConfig = merge(common, {
   entry: "./src/index.js",
   output: {
     filename: "grain-runner-browser.js",
-    path: __dirname + "/dist",
```

Comment:
The migration guide recommended removing this if it matched the default (ours does)